src/RelationParser.h

```diff
@@ -15,7 +15,7 @@
 namespace snowcrash {
 
     /** Link Relation matching regex */
-    const char* const RelationRegex = "^[[:blank:]]*[Rr]elation[[:blank:]]*:" SYMBOL_IDENTIFIER "?$";
+    const char* const RelationRegex = "^[[:blank:]]*[Rr]elation[[:blank:]]*:[[:blank:]]*([a-z][a-z0-9.-]*)?[[:blank:]]*$";
```

Comment:
I suggest to use define for `*([a-z][a-z0-9.-]*)?`. Same practice is used througth full project